frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -297,140 +431,40 @@ const ResourceDescription = ({ resource }: { resource?: LearningResource }) => {
-        return prev.start_date &&
-          current.start_date &&
-          Date.parse(prev.start_date) > now &&
-          Date.parse(prev.start_date) - now <
-            Date.parse(current.start_date) - now
-          ? prev
-          : current
-      }, resource!.runs![0])
-      setSelectedRun(closest)
-    }
-  }, [resource])
-
-  const onDateChange: SimpleSelectProps["onChange"] = (event) => {
-    const run = resource?.runs?.find(
-      (run) => run.id === Number(event.target.value),
-    )
-    if (run) setSelectedRun(run)
-  }
-
-  const DateSection = ({ hide }: { hide?: boolean }) => {
-    if (hide) {
-      return null
-    }
-    if (!resource) {
-      return <Skeleton height={40} style={{ marginTop: 0, width: "60%" }} />
-    }
-
-    const dateOptions: SimpleSelectProps["options"] =
-      resource.runs
-        ?.sort((a, b) => {
-          if (a?.start_date && b?.start_date) {
-            return Date.parse(a.start_date) - Date.parse(b.start_date)
-          }
-          return 0
-        })
-        .map((run) => {
-          return {
-            value: run.id.toString(),
-            label: formatRunDate(run, asTaughtIn),
-          }
-        }) ?? []
-
-    if (
-      [ResourceTypeEnum.Course, ResourceTypeEnum.Program].includes(
-        resource.resource_type as "course" | "program",
-      ) &&
-      multipleRuns
-    ) {
-      return (
-        <DateContainer>
-          <DateLabel>{label}</DateLabel>
-          <SimpleSelect
-            value={selectedRun?.id.toString() ?? ""}
-            onChange={onDateChange}
-            options={dateOptions}
-          />
-        </DateContainer>
-      )
-    }
-
-    if (!selectedRun) return <NoDateSpacer />
-
-    const formatted = formatRunDate(selectedRun, asTaughtIn)
-    if (!formatted) {
-      return <NoDateSpacer />
-    }
-    return (
-      <DateSingle>
-        <DateLabel>{label}</DateLabel>
-        {formatted ?? ""}
-      </DateSingle>
-    )
-  }
-
-  const isVideo =
-    resource &&
-    (resource.resource_type === ResourceTypeEnum.Video ||
-      resource.resource_type === ResourceTypeEnum.VideoPlaylist)
-
   return (
-    <Container padTop={isVideo}>
-      <DateSection hide={isVideo} />
-      <ImageSection resource={resource} config={imgConfig} />
-      <CallToActionSection resource={resource} hide={isVideo} />
-      <DetailSection resource={resource} />
-      <InfoSection
+    <>
+      <TitleSection
         resource={resource}
-        run={selectedRun}
-        user={user}
-        onAddToLearningPathClick={onAddToLearningPathClick}
-        onAddToUserListClick={onAddToUserListClick}
+        closeDrawer={closeDrawer ?? (() => {})}
       />
-    </Container>
+      <Container>
+        <ContentContainer>
+          <LeftContainer>
+            <DetailSection resource={resource} />
+            <InfoSection resource={resource} />
+          </LeftContainer>
+          <RightContainer>
+            <CallToActionSection
```

Comment:
This adds a call to action for all resource types. The button does not go anywhere for video playlists and learning paths since url is null for those resources and getCallToActionUrl only has special logic for podcasts. http://api.open.odl.local:8063/api/v1/learning_resources_search/?resource_type=video_playlist . You can add special logic to construct the correct url based on the channel_id for the video playlist. I'm not sure if we are supposed to have learning paths in prod. There is currently one in the production search but I think that it might not be supposed to be visible